README.md

```diff
@@ -26,24 +53,170 @@ resource "aws_db_instance" "default" {
   username             = "foo"
   password             = "foobarbaz"
   parameter_group_name = "default.mysql5.7"
-  apply_immediately = "true"
-  skip_final_snapshot = "true"
+  apply_immediately    = "true"
+  skip_final_snapshot  = "true"
 }
 
 module "rds_alarms" {
-  source = "github::https://github.com/bitflight-public/terraform-aws-rds-alerts.git?ref=master"
+  source         = "github::https://github.com/bitflight-public/terraform-aws-rds-alerts.git?ref=master"
   db_instance_id = "${aws_db_instance.default.id}"
 }
 ```
 
-## Variables
-| Name                         | Description                                                                              | Required |
-|------------------------------|------------------------------------------------------------------------------------------|----------|
-| db_instance_id               | The instance ID of the RDS database instance that you want to monitor.               		| Yes 		 |
-| burst_balance_threshold      | The minimum percent of General Purpose SSD (gp2) burst-bucket I/O credits available. 		| No       |
-| cpu_utilization_threshold    | The maximum percentage of CPU utilization.                            										| No       |
-| cpu_credit_balance_threshold | The minimum number of CPU credits (t2 instances only) available. 												| No       |
-| disk_queue_depth_threshold   | The maximum number of outstanding IOs (read/write requests) waiting to access the disk. 	| No       |
-| freeable_memory_threshold    | The minimum amount of available random access memory in Byte. 														| No       |
-| free_storage_space_threshold | The minimum amount of available storage space in Byte. 																	| No       |
-| swap_usage_threshold         | The maximum amount of swap space used on the DB instance in Byte. 												| No       |
\ No newline at end of file
+
+
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| burst_balance_threshold | The minimum percent of General Purpose SSD (gp2) burst-bucket I/O credits available. | string | `20` | no |
+| cpu_credit_balance_threshold | The minimum number of CPU credits (t2 instances only) available. | string | `20` | no |
+| cpu_utilization_threshold | The maximum percentage of CPU utilization. | string | `80` | no |
+| db_instance_id | The instance ID of the RDS database instance that you want to monitor. | string | - | yes |
```

Comment:
use `` (instead of `-`) as default empty string?

Reply:
`terraform-docs` writes the `-`